lib/foreman_tasks/tasks/generate_task_actions.rake

```diff
@@ -0,0 +1,18 @@
+#
+# generate_task_actions.rake is a data migration task to properly fill the missing
+# values into the foreman_tasks_tasks table.
+#
+# Run "foreman-rake foreman_tasks:generate_task_actions" to generate missing values.
+
+namespace :foreman_tasks do
+  desc 'Generate missing values for action column in foreman_tasks_tasks table.'
+  task :generate_task_actions => :environment do
+    scope = ::ForemanTasks::Task.where(:action => nil)
+
+    puts "Generating action for #{scope.count} tasks."
+    scope.find_each do |task|
+      task.action = task.format_input
```

Comment:
Some info about the progress would be nice. Let's say printing `done/total', every 100 records or something like that, to give the user some idea on when to expect it will finish.